e2e/AddItemTest.spec.ts

```diff
@@ -0,0 +1,23 @@
+import { test, expect } from '@playwright/test';
+import 'dotenv/config';
+
+test('Add item to the cart', async ({ page }) => {
+   await page.goto('https://localhost:19888/');
```

Comment:
add 'https://localhost:19888/' to baseURL in playwright config and use a '/' here.